Docker_files/ocsci_container/README.md

```diff
@@ -2,54 +2,120 @@
 
 This project builds containers for [ocs-ci].
 The primary purpose for doing so was to be able to run [ocs-ci] tests with container
-
 The containers are expected to run with a service account that has admin credentials.
 
 ### Build Image
-
-**Change directory to Docker_files/ocsci_container [$ cd Docker_files/ocsci_container]**
-
-**docker/podman build -t <image-name> -f Dockerfile_ocsci . --build-arg BRANCH_ID_ARG=<branch-id/master/stable>**
-
-BRANCH_ID_ARG:
-There are 3 options:
-1. BRANCH_ID_ARG is empty -> checkout to stable branch
-2. BRANCH_ID_ARG=master -> checkout to master branch
-3. BRANCH_ID_ARG=PR-ID [1234] -> checkout to relevant branch based on pr-id
-
-### Run Container
-```commandline
-docker/podman run -v <kubeconfig-path>:/opt/cluster <image-name> --ocp-version <ocp-version>
---ocs-version <ocs-version> --cluster-name <cluster-name> <test-path>
-```
-*Add Params:
-```
-kubeconfig-path: Path to kubeconfig on your local machine
-image-name: Image name
-run-ci params: without "--cluster-path"
-**Don't need to add "run-ci" string to docker/podman run cmd
-
-```
-
-Example:
-
-```commandline
-docker run -v ~/ClusterPath:/opt/cluster ocsci_image --ocp-version 4.12 --ocs-version 4.12
---cluster-name cluster-name tests/manage/z_cluster/test_must_gather.py
-```
+* Build Image on your local machine
+* PARAMS:
+  * BRANCH - branch name [e.g: release-4.12]
+  * IMAGE_NAME - image name [e.g. quay.io/ocsci/ocs-ci-container:release-4.12]
+  * ENGINE - engine name [podman/docker]
+* Example:
+  ```
+  make build-image \
+  BRANCH=release-4.12 \
+  IMAGE_NAME=quay.io/ocsci/ocs-ci-container:release-4.12 \
+  ENGINE=podman
+  ```
+
+### Running ODF tests
+* Running ODF tests
+* PARAMS:
+  * CLUSTER_PATH - path to kubeconfig on your local machine [e.g: ~/cluster_path]
+  * RUN_CI - run-ci cmd for more info https://github.com/red-hat-storage/ocs-ci/tree/master/docs
+    ```
+    run-ci --cluster-path /opt/cluster \
+    --ocp-version 4.12 \
+    --ocs-version 4.12 \
+    --cluster-name cluster-name \
+     tests/manage/z_cluster/test_osd_heap_profile.py
+    ```
+  * ENGINE - engine name [podman/docker] [optional, by default docker]
+  * PULL_IMAGE - there is option to pull the image from `quay.io/ocsci/ocs-ci-container`
+  * IMAGE_NAME - image name [e.g. quay.io/ocsci/ocs-ci-container:release-4.12]
+  * AWS_PATH - PATH to AWS credintials [e.g = ~/.aws] [optional]
+* Example:
+  ```
+  make run-odf CLUSTER_PATH=~/ClusterPath \
+  RUN_CI="run-ci .." \
```

Comment:
If I understand it correctly, the two dots `..` might represent the rest of the parameters for the run-ci command, right? 
I would propose to use three dots `...`, to not confuse it with _"parent directory"_.